app/src/main/java/mn/devfest/api/model/Session.java

```diff
@@ -102,6 +113,11 @@ public String getCategory() {
     }
 
     public DateTime getStartTime() {
-        return startTime;
+        try {
+            return DateTime.parse(startTime);
+        } catch (Exception e) {
+            Timber.e(e, "Failed to parse the startTime");
+            return new DateTime();
+        }
```

Comment:
I had an idea at work today that I'm trying now.
I'm going to try just having a separate field for the DateTime and lazyloading it in it's accessor method.